auth0/src/main/java/com/auth0/android/authentication/storage/CredentialsManager.java

```diff
@@ -97,6 +97,32 @@ public void onFailure(AuthenticationException error) {
         });
     }
 
+    /**
+     * Checks if a valid pair of credentials can be obtained from this manager.
+     */
+    public boolean hasCredentials() {
```

Comment:
I'd mention that they are valid credentials not only he existence of one. Also I'd reflect that in the name